resources/views/docs/info.blade.php

```diff
@@ -25,6 +25,9 @@
 
 ## Breaking Changes
 
+### 2022-11-11
+- `recent_messages` in [ChatChannel](#chatchannel) has been deprecated, it will be removed from [Create Channel] (#create-channel) response in the near future.
```

Comment:
```suggestion
- `recent_messages` in [ChatChannel](#chatchannel) has been deprecated, it will be removed from [Create Channel](#create-channel) response in the near future.
```